options/locale/locale_de-DE.ini

```diff
@@ -431,6 +431,7 @@ migrate.clone_local_path=oder Pfad auf dem lokalen Server
 migrate.permission_denied=Ihnen fehlen die Rechte zum Importieren lokaler Repositories.
 migrate.invalid_local_path=Der lokale Pfad ist ungültig, existiert nicht oder ist kein Ordner.
 migrate.failed=Fehler bei Migration: %v
+migrate.lfs_mirror_unsupported = Migrieren von LFS Objekten wird nicht unterstützt - verwenden Sie stattdessen 'git lfs fetch --all' und 'git lfs push --all'.
```

Comment:
Strings for langs other than English should not be added directly, but via [Crowdin](https://crowdin.com/project/gitea)